test/e2e/set/nodeconfig/nodeconfig_disksetup.go

```diff
@@ -548,6 +523,168 @@ var _ = g.Describe("Node Setup", framework.Serial, func() {
 			},
 		}),
 	)
+
+	g.It("should successfully start a failed mount unit with a corrupted filesystem when it's overwritten with a clean one", func(ctx context.Context) {
+		o.Expect(matchingNodes).NotTo(o.BeEmpty())
+		o.Expect(matchingNodes[0]).NotTo(o.BeNil())
+		nodeUnderTest := matchingNodes[0]
+
+		devicePath := "/dev/loops/disk"
```

Comment:
lets use unique device names, for example a f.Namespace(), to not collide with other tests

Reply:
done